app/assets/javascripts/event_staff.js

```diff
@@ -1,23 +1,75 @@
 $(document).ready(function(){
-  $(".add_staff_field").autocomplete({
-    delay: 500,
-    minLength: 4,
-    source: "/lookup_staffs",
-    select: function(event, ui){
-      var staffId = ui.item.data;
-      if ($(".event_staffs").length > 0){
-        $.get("/user_info/" + staffId, function(data){
-          $(".event_staffs").append(data);
-          $(".add_staff_field").val("");
-        });
-      }
-      else {
-          $("#new_staff").val(staffId);
+
+  var isExistingOnPage = function (userId) {
+    var parent = $("#event_staffs").children(),
+        isOnPage = false;
+
+    for (var i = 0; i < parent.length; i += 1) {
+      if(parent[i].dataset.id === userId) {
+      isOnPage = true;
+      break;
       }
     }
+    return isOnPage;
+  };
+
+  var generateHtml = function(data){
+    return `<div class="chip" data-id= "` + data.user_id + `">
+      <img src="`+ data.profile_url + `" alt="Contact Person">
+      ` + data.first_name + ` (` + data.user_role+ `)&emsp;
+      <a href="#" data-remote="true"><span class="remove_staff">x</span></a>
```

Comment:
Expected an identifier and instead saw '<'.<br>Unrecoverable syntax error. (26% scanned).